src/snowflake/connector/cursor.py

```diff
@@ -1314,7 +1314,16 @@ def fetch_pandas_batches(self, **kwargs: Any) -> Iterator[DataFrame]:
         return self._result_set._fetch_pandas_batches(**kwargs)
 
     def fetch_pandas_all(self, **kwargs: Any) -> DataFrame:
-        """Fetch Pandas dataframes in batches, where 'batch' refers to Snowflake Chunk."""
+        """
+        Fetch Pandas dataframes in batches, where 'batch' refers to Snowflake Chunk.
+
+        Returns:
+            A pandas dataframe containing the cursors query results.
+
+        Note:
+            Timestamp types that are nested in structured types may not be accurately represented
+            due to a limitation in the pyarrow to pandas conversion.
```

Comment:
I'm wondering is there a link we can provide to which points to a well-known issue or official doc from pyarrow about this part?